### PR TITLE
Add Support For Symfony 7.0

### PR DESCRIPTION
**Pull request by Galironfydar**

### CODE REVIEW

File: src/Services/DbService.php [hunk 1]
```diff
@@ -156,4 +156,14 @@ public function getDefaultTenantDataBase(): TenantDbConfigurationInterface
...
     }
+
+    public function getAllTenantDataBases(): array
+    {
+        return $this->entityManager->getRepository($this->tenantDbListEntity)->findAll();
+    }
+
+    public function getTenantDataBase(int $id): ?TenantDbConfigurationInterface
+    {
+        return $this->entityManager->getRepository($this->tenantDbListEntity)->findOneBy(['dbName' => $databaseName]);
```
Comment (RamyHakam) [inline src/Services/DbService.php:167]: You do not use `$id` in this function, Also `$databaseName` not exist

**Comment by RamyHakam** [comment]
Hey @Galironfydar  Thanks for your PR 
I have some general comments on your PR 
1. Can you Please split your PR in 2, Cuz this PR has 2 different tasks For example you can split them to :
      **One for Symfony 7 Support**
     **One for Delete Comment** 
2. Deleting tenant DB only will not work  in this approach you need to have a safe clean 
, Cuz it has a relation with the tenant user config which is the tenant user for this database 
So If you want to add this command you need to delete the tenant user config and then delete the db config 
Please see the example repo for that [multi-tenancy-project-example](https://github.com/RamyHakam/multi-tenancy-project-example)
Also deleting Tenant DB is a critical command that should not be available on the prod 
you can restrict this command only on the dev env use this function in your command 
`   public function isEnabled(): bool
    {
        return 'dev' === getenv('APP_ENV');
    }`

**Review by RamyHakam — CHANGES_REQUESTED** [review]

**Comment by Galironfydar** [comment]
Apologies, the tenant deletion stuff was me playing around, I'll create another pull request for that, I've just removed it.

**Comment by NikoGrano** [comment]
@RamyHakam Changes look good to me, would you mind merging and creating the release for the composer? This is currently a blocker for me to upgrade 7.x.

Thanks for all your work!

**Review by NikoGrano — APPROVED** [review]

**Review by RamyHakam — APPROVED** [review]

**Pull request merged**